File: Java/Fibonacci.java
import java.util.Scanner;
import java.util.Arrays;

public class Fibonacci {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.println("Enter the limit: ");
		int n = scan.nextInt();
		int[] result = new int[n];
		for ( int i=0;i<n;i++ ) {
			result[i] = fibonacci(i);
		}
		System.out.println("First "+n+" element of fibonacci sequence are :"+Arrays.toString(result));
	}

	public static int fibonacci(int n) {
		if ( n==1 || n==0 ) {
			return 1;
		}
		int ans = fibonacci(n-1) + fibonacci(n-2);
		return ans;
	}
}

File: Python/fibonacci.py
def fibonacci(n):
	if n == 1 or n == 0 :
		return 1
	ans = fibonacci(n-1) + fibonacci(n-2)
	return ans

# # Tail recursion fibonacci
# def fibonacci(n,a = 0,b = 1):
#     if n == 0:
#         return a
#     if n == 1:
#         return b
#     return fibonacci(n-1,b,a+b)


if __name__ == '__main__':
	n = raw_input("Enter the limit: ")
	# result = fibonacci(int(n))
	result = []
	for i in range(0,int(n)) :
		result.append(fibonacci(i))
	print 'First ',n,' element of fibonacci sequence are :',result